ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/OptimizingPlanner.java

```diff
@@ -117,7 +117,6 @@ public List<TaskDescriptor> planTasks() {
 
     List<PartitionEvaluator> evaluators = new ArrayList<>(partitionPlanMap.values());
     evaluators.sort(Comparator.comparing(PartitionEvaluator::getWeight));
-    Collections.reverse(evaluators);
```

Comment:
What is the reason for changing the sorting from descending to ascending by Weight?

Reply:
I think prioritizing partitions with low cost will free up the optimizer's resources as soon as possible, plus it will try to process as many partitions as possible in one more process.

Reply:
To my knowledge, we prioritize the partitions with high cost to avoid starvation. 
This is because the cost of a partition is expected to decrease after optimizing it. Therefore, if we prioritize partitions with low cost, it may cause high cost partitions to never optimizing.

Reply:
Considering this risk, I suggest keeping the original sorting order. What do you think?

Reply:
Make sense